curator-framework/src/test/java/org/apache/curator/framework/imps/TestFramework.java

```diff
@@ -50,26 +59,24 @@
 import org.apache.zookeeper.Watcher;
 import org.apache.zookeeper.ZooDefs;
 import org.apache.zookeeper.ZooKeeper;
+import org.apache.zookeeper.client.ZKClientConfig;
 import org.apache.zookeeper.data.ACL;
 import org.apache.zookeeper.data.Stat;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.BlockingQueue;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.LinkedBlockingQueue;
-import java.util.concurrent.TimeUnit;
+import com.google.common.collect.Lists;
 
 @SuppressWarnings("deprecation")
 @Tag(CuratorTestBase.zk35TestCompatibilityGroup)
 public class TestFramework extends BaseClassForTests
 {
+	private final Logger log = LoggerFactory.getLogger(getClass());
```

Comment:
Usually loggers are static fields
`private static final Logger log = LoggerFactory.getLogger(TestFramework.class);`

Reply:
I wrote it exactly as already implemented at the same package, to be consistent:

https://github.com/apache/curator/blob/b9dc9bd4056dba18c29a4f3127df43b930449abe/curator-framework/src/test/java/org/apache/curator/framework/imps/TestFrameworkBackground.java#L56

https://github.com/apache/curator/blob/b9dc9bd4056dba18c29a4f3127df43b930449abe/curator-framework/src/test/java/org/apache/curator/framework/imps/TestFrameworkEdges.java#L78

Reply:
sorry.
you are right.
I forgot that we are in Curator and we have different conventions.

thanks for checking out